### DISCUSSION TITLE
Tronity import

### DISCUSSION THREAD

**Discussion by PaulNokel** [teslamate-org/teslamate#2662] [Ideas]
Hi,

I was wondering if there are any ideas around the topic of importing data from Tronity to Teslamate. Right now I am not even able to export my Tronity data due to technical issues, but I'd really love to be able to move my ~7 month of historical data to Teslamate.

Best regards,
Paul

**Comment by matthieu-labas** [comment]
I'd love to be able to import data from external sources too (and [asked about it](https://github.com/adriankumpf/teslamate/discussions/2694) too).

**Comment by endurance0** [comment]
Hello,

Any Update to this topic?
I‘m currently using Tronity. But now I‘ll get a second Tesla, and I‘ll change the App.
